src/containers/InternalContactOverview.js

```diff
@@ -312,6 +81,14 @@ class InternalContactOverview extends PureComponent {
             width: '50%',
           }}
         >
+        <SearchBar
+          onChange={() => console.log('onChange')}
```

Comment:
hmm maybe this is better than the one I made after all haha how does this one look? and does it work the same way?

Reply:
Picture of Search Bar 

![search bar](https://user-images.githubusercontent.com/35805088/38917272-8fd110fe-42ea-11e8-8d2c-6de5ce660ec2.png)